content/en/blog/_posts/2024-08-15-kubernetes-1.31-image-volume-source.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: blog
+title: "Kubernetes 1.31: Read Only Volumes Based On OCI Artefacts (alpha)"
```

Comment:
```suggestion
title: "Kubernetes 1.31: Read Only Volumes Based On OCI Artifacts (alpha)"
```

Reply:
@sftim suggested this spelling, so I’d assume it’s preferred?

Reply:
Whatever the folks in the US would write. I'm from England, so I natively speak a different flavo[u]r English to the folks in the USA, and sometimes I get it wrong.

Reply:
Alright, changed as suggested :+1: